src/ZfcRbac/Service/AuthorizationService.php

```diff
@@ -88,7 +112,10 @@ public function setAssertion($permission, $assertion)
      */
     public function setAssertions(array $assertions)
     {
-        $this->assertions = $assertions;
```

Comment:
don't know if this is good creating assertions while setting them. Maybe we could create AssertionSet only if we need in `assert()` method?